service/controller/v24/resource/cloudformation/delete.go

```diff
@@ -31,6 +32,13 @@ func (r *Resource) ApplyDeleteChange(ctx context.Context, obj, deleteChange inte
 			return microerror.Mask(err)
 		}
 
+		err = disableMasterTerminationProtection(ctx, key.MasterInstanceName(customObject))
+		if err != nil {
+			return microerror.Mask(err)
+		}
+
+		r.logger.LogCtx(ctx, "level", "debug", "message", "disabled master instance termination protection")
+
 		stackName := aws.String(key.MainGuestStackName(customObject))
 
 		updateTerminationProtection := &cloudformation.UpdateTerminationProtectionInput{
```

Comment:
What is this doing then?

Reply:
One is for the cloud formation stack and the other for the master instance (EC2 instance)

Reply:
💡